move releases to root level

lorerod opened the pull request:
<!--
Please use semantic PR titles that respect this format:

<type>(#<issue number>): <subject>

Quick example:

feat(#1234): add hat wobble
^--^(#^--^): ^------------^
|     |      |
|     |      + - > subject
|     |
|     + -------- > issue number
|
+ -------------- > type: chore, feat, fix, perf.

https://docs.communityhealthtoolkit.org/contribute/code/workflow/#commit-message-format
-->

# Description

move releases to root level and update references

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

mrjones-plip commented:
sorry for the delay in tending to this @lorerod ! I'll get to in the coming day or two.

mrjones-plip commented on content/en/releases/core/_index.md:
```diff
@@ -0,0 +1,238 @@
+description: >
+  Versions currently supported, dependencies, and release notes for the CHT Core Framework
+no_list: true
+aliases:
+  -    /core/overview/supported-software
```
We'll need to add aliases to ever page to be redirected from the old URL to the new URL.  Here's an example on this page.  It's original URL was` https://docs.communityhealthtoolkit.org/core/releases/` but the new URL is `https://docs.communityhealthtoolkit.org/releases/core/`, so we'll add a new alias like this:

```suggestion
aliases:
  -    /core/releases
  -    /core/overview/supported-software
```

mrjones-plip reviewed (changes requested):
This is a LOT Of pages to move - Nice work on this!  I kinda feel like the `/core/` sub-directory is a little overkill as we don't have CHT Sync or Watchdog releases in `/sync/` or `/watchdog/` directories , but I think we can make this move at another time if we feel strongly.

The big fix is that we need to add an `alias:` for every page we moved.  To test these are there, I checked out your branch and deleted my hugo container and relaunched it.  This cleaned my cache.  If you run `hugo` on the CLI, call `hugo --cleanDestinationDir` before starting it to achieve the same.

I then ran the `./.github/scripts/check.urls.sh` to get the output below.  If you merge `main` to your branch the four `404`s in the `/hosting`   dir will be fixed.

I made one suggestion as an example that needs to be apply to all `404`s below for the the `/core/release` URLs.

```
$ ./.github/scripts/check.urls.sh

Are you on a test branch and is hugo running on http://localhost:1313 ?

"y" to proceed, "n" to cancel y

Fetching URLs from production.

Checking 441 URLs, be patient.  Any non 200 URLs will be listed here:

http://localhost:1313/core/releases/ 404
http://localhost:1313/hosting/4.x/production/kubernetes/gcp-multinode/ 404
http://localhost:1313/hosting/couch2pg/setup-and-devlopment/ 404
http://localhost:1313/hosting/couch2pg/migration/ 404
http://localhost:1313/hosting/couch2pg/ 404
http://localhost:1313/core/releases/0.4.15-and-earlier/ 404
http://localhost:1313/core/releases/2.10.0/ 404
http://localhost:1313/core/releases/2.10.1/ 404
http://localhost:1313/core/releases/2.10.2/ 404
http://localhost:1313/core/releases/2.10.3/ 404
http://localhost:1313/core/releases/2.11.0/ 404
http://localhost:1313/core/releases/2.11.1/ 404
http://localhost:1313/core/releases/2.11.2/ 404
http://localhost:1313/core/releases/2.11.3/ 404
http://localhost:1313/core/releases/2.12.0/ 404
http://localhost:1313/core/releases/2.12.1/ 404
http://localhost:1313/core/releases/2.12.2/ 404
http://localhost:1313/core/releases/2.12.3/ 404
http://localhost:1313/core/releases/2.12.4/ 404
http://localhost:1313/core/releases/2.12.5/ 404
http://localhost:1313/core/releases/2.13.0/ 404
http://localhost:1313/core/releases/2.13.1/ 404
http://localhost:1313/core/releases/2.13.2/ 404
http://localhost:1313/core/releases/2.13.3/ 404
http://localhost:1313/core/releases/2.13.4/ 404
http://localhost:1313/core/releases/2.13.5/ 404
http://localhost:1313/core/releases/2.13.6/ 404
http://localhost:1313/core/releases/2.13.7/ 404
http://localhost:1313/core/releases/2.14.0/ 404
http://localhost:1313/core/releases/2.14.1/ 404
http://localhost:1313/core/releases/2.14.2/ 404
http://localhost:1313/core/releases/2.14.3/ 404
http://localhost:1313/core/releases/2.15.0/ 404
http://localhost:1313/core/releases/2.16.0/ 404
http://localhost:1313/core/releases/2.16.1/ 404
http://localhost:1313/core/releases/2.17.0/ 404
http://localhost:1313/core/releases/2.18.0/ 404
http://localhost:1313/core/releases/2.18.1/ 404
http://localhost:1313/core/releases/2.6.0/ 404
http://localhost:1313/core/releases/2.6.1/ 404
http://localhost:1313/core/releases/2.6.2/ 404
http://localhost:1313/core/releases/2.6.3/ 404
http://localhost:1313/core/releases/2.7.0/ 404
http://localhost:1313/core/releases/2.7.1/ 404
http://localhost:1313/core/releases/2.7.2/ 404
http://localhost:1313/core/releases/2.7.3/ 404
http://localhost:1313/core/releases/2.8.0/ 404
http://localhost:1313/core/releases/2.8.1/ 404
http://localhost:1313/core/releases/2.8.2/ 404
http://localhost:1313/core/releases/2.8.3/ 404
http://localhost:1313/core/releases/2.8.4/ 404
http://localhost:1313/core/releases/2.8.5/ 404
http://localhost:1313/core/releases/2.9.0/ 404
http://localhost:1313/core/releases/2.9.1/ 404
http://localhost:1313/core/releases/3.0.0/ 404
http://localhost:1313/core/releases/3.1.0/ 404
http://localhost:1313/core/releases/3.10.0/ 404
http://localhost:1313/core/releases/3.10.1/ 404
http://localhost:1313/core/releases/3.10.2/ 404
http://localhost:1313/core/releases/3.10.3/ 404
http://localhost:1313/core/releases/3.10.4/ 404
http://localhost:1313/core/releases/3.10.5/ 404
http://localhost:1313/core/releases/3.11.0/ 404
http://localhost:1313/core/releases/3.11.1/ 404
http://localhost:1313/core/releases/3.11.2/ 404
http://localhost:1313/core/releases/3.11.3/ 404
http://localhost:1313/core/releases/3.12.0/ 404
http://localhost:1313/core/releases/3.12.1/ 404
http://localhost:1313/core/releases/3.13.0/ 404
http://localhost:1313/core/releases/3.14.0/ 404
http://localhost:1313/core/releases/3.14.1/ 404
http://localhost:1313/core/releases/3.14.2/ 404
http://localhost:1313/core/releases/3.15.0/ 404
http://localhost:1313/core/releases/3.16.0/ 404
http://localhost:1313/core/releases/3.16.1/ 404
http://localhost:1313/core/releases/3.17.0/ 404
http://localhost:1313/core/releases/3.17.1/ 404
http://localhost:1313/core/releases/3.17.2/ 404
http://localhost:1313/core/releases/3.2.0/ 404
http://localhost:1313/core/releases/3.2.1/ 404
http://localhost:1313/core/releases/3.3.0/ 404
http://localhost:1313/core/releases/3.4.0/ 404
http://localhost:1313/core/releases/3.4.1/ 404
http://localhost:1313/core/releases/3.5.0/ 404
http://localhost:1313/core/releases/3.6.0/ 404
http://localhost:1313/core/releases/3.6.1/ 404
http://localhost:1313/core/releases/3.6.2/ 404
http://localhost:1313/core/releases/3.7.0/ 404
http://localhost:1313/core/releases/3.7.1/ 404
http://localhost:1313/core/releases/3.8.0/ 404
http://localhost:1313/core/releases/3.8.1/ 404
http://localhost:1313/core/releases/3.8.2/ 404
http://localhost:1313/core/releases/3.9.0/ 404
http://localhost:1313/core/releases/3.9.1/ 404
http://localhost:1313/core/releases/3.9.2/ 404
http://localhost:1313/core/releases/4.0.0/ 404
http://localhost:1313/core/releases/4.0.1/ 404
http://localhost:1313/core/releases/4.1.0/ 404
http://localhost:1313/core/releases/4.1.1/ 404
http://localhost:1313/core/releases/4.1.2/ 404
http://localhost:1313/core/releases/4.10.0/ 404
http://localhost:1313/core/releases/4.11.0/ 404
http://localhost:1313/core/releases/4.12.0/ 404
http://localhost:1313/core/releases/4.13.0/ 404
http://localhost:1313/core/releases/4.14.0/ 404
http://localhost:1313/core/releases/4.15.0/ 404
http://localhost:1313/core/releases/4.16.0/ 404
http://localhost:1313/core/releases/4.17.0/ 404
http://localhost:1313/core/releases/4.18.0/ 404
http://localhost:1313/core/releases/4.2.0/ 404
http://localhost:1313/core/releases/4.2.1/ 404
http://localhost:1313/core/releases/4.2.2/ 404
http://localhost:1313/core/releases/4.2.3/ 404
http://localhost:1313/core/releases/4.2.4/ 404
http://localhost:1313/core/releases/4.3.0/ 404
http://localhost:1313/core/releases/4.3.1/ 404
http://localhost:1313/core/releases/4.3.2/ 404
http://localhost:1313/core/releases/4.4.0/ 404
http://localhost:1313/core/releases/4.4.1/ 404
http://localhost:1313/core/releases/4.4.2/ 404
http://localhost:1313/core/releases/4.5.0/ 404
http://localhost:1313/core/releases/4.5.1/ 404
http://localhost:1313/core/releases/4.5.2/ 404
http://localhost:1313/core/releases/4.6.0/ 404
http://localhost:1313/core/releases/4.7.0/ 404
http://localhost:1313/core/releases/4.7.1/ 404
http://localhost:1313/core/releases/4.7.2/ 404
http://localhost:1313/core/releases/4.8.0/ 404
http://localhost:1313/core/releases/4.8.1/ 404
http://localhost:1313/core/releases/4.9.0/ 404
Successfully checked 441 URLs!
```

lorerod commented:
Thanks, @mrjones-plip! I agree—this is a big move, and I appreciate the review. Good call on the `/core/` sub-directory naming. I'm happy to revisit the structure later if it feels too much now.

I went ahead and implemented all the suggested changes:
- I removed the `/core/` subdirectory — all CHT Core release notes are now under `/releases/`.
- I added aliases for all moved pages to maintain backward compatibility.

I also merged in main to take care of the 404s from the `/hosting/` directory, and appreciated the tip on running `hugo --cleanDestinationDir `. I was missing this step to make everything work properly.

Additionally, I updated `check.urls.sh` to be compatible with macOS. I realized it wasn't working correctly for me. That said, I’m wondering if those commits might be better off in a separate PR. I'm happy to split them out if that’s preferred. Let me know what you think.

All updates have been pushed! Could you give it another review when you have time?

mrjones-plip commented on content/en/releases/_index.md:
```diff
@@ -0,0 +1,239 @@
+---
+title: "Releases"
+linkTitle: "Core Framework"
```
I think this should be "Releases" as the left hand nav is a bit confusing with so many "CHT"s and "Core"s:

```suggestion
linkTitle: "Releases"
```

![image](https://github.com/user-attachments/assets/c17613f3-92f9-485d-8dfb-8ae7fbd6956e)

mrjones-plip commented on .github/scripts/check.urls.sh:
```diff
@@ -56,8 +63,6 @@ if [[ ! $REPLY =~ ^[nN]$ ]]
 then
     echo;echo "Fetching URLs from production."
     prod_urls=$(get_urls_from_prod_site_map)
-    url_count=$(echo "$prod_urls" | wc -l | cut -d' ' -f1)
-    echo;echo "Checking ${url_count} URLs, be patient.  Any non 200 URLs will be listed here:"
-    run_checks "$prod_urls"
+    url_count=$(run_checks "$prod_urls")
     echo "Successfully checked ${url_count} URLs!"
```
The way the this change makes the output happen there's `Successfully checked` then all the URLs and status then  `438 URLs!` See below for an example.

can you refactor this so that either `run_checks()` handles the return values correctly (count vs URL status) or break out the URL count to be separate like it was before?


```
➜  cht-docs git:(move-releases) ./.github/scripts/check.urls.sh 

Are you on a test branch and is hugo running on http://localhost:1313 ?

"y" to proceed, "n" to cancel y

Fetching URLs from production.
Successfully checked 
http://localhost:1313/core/releases/4.5.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.5.0/ 200 
http://localhost:1313/core/releases/4.5.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.5.1/ 200 
http://localhost:1313/core/releases/4.5.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.5.2/ 200 
438 URLs!
```

mrjones-plip replied:
Oh!  Looking a bit closer, the output should actually show the URL count twice so the user know how long to wait. Everything up to `will be listed here:` is shown instantly, and then the rest is showing errors as it goes and finally `Successfully checked 438 URLs!` when done:


```
$ ./.github/scripts/check.urls.sh    

Are you on a test branch and is hugo running on http://localhost:1313 ?

"y" to proceed, "n" to cancel y

Fetching URLs from production.

Checking 438 URLs, be patient.  Any non 200 URLs will be listed here:

Successfully checked 438 URLs!
```

lorerod replied:
Sorry about this! I didn't see it. I have a new version that for me is showing this format:

```
./.github/scripts/check.urls.sh

Are you on a test branch and is hugo running on http://localhost:1313 ?

"y" to proceed, "n" to cancel y

Fetching URLs from production.
Found 439 URLs to check, be patient. Any non 200 URLs will be listed here:

http://localhost:1313/reference-apps/anc/ 404
http://localhost:1313/core/releases/ Is redirected! Result is:
    -> http://localhost:1313/releases/ 200 
http://localhost:1313/reference-apps/ 404
http://localhost:1313/core/releases/0.4.15-and-earlier/ Is redirected! Result is:
    -> http://localhost:1313/releases/0.4.15-and-earlier/ 200 
http://localhost:1313/core/releases/2.10.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.10.0/ 200 
http://localhost:1313/core/releases/2.10.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.10.1/ 200 
http://localhost:1313/core/releases/2.10.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.10.2/ 200 
http://localhost:1313/core/releases/2.10.3/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.10.3/ 200 
http://localhost:1313/core/releases/2.11.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.11.0/ 200 
http://localhost:1313/core/releases/2.11.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.11.1/ 200 
http://localhost:1313/core/releases/2.11.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.11.2/ 200 
http://localhost:1313/core/releases/2.11.3/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.11.3/ 200 
http://localhost:1313/core/releases/2.12.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.12.0/ 200 
http://localhost:1313/core/releases/2.12.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.12.1/ 200 
http://localhost:1313/core/releases/2.12.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.12.2/ 200 
http://localhost:1313/core/releases/2.12.3/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.12.3/ 200 
http://localhost:1313/core/releases/2.12.4/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.12.4/ 200 
http://localhost:1313/core/releases/2.12.5/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.12.5/ 200 
http://localhost:1313/core/releases/2.13.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.13.0/ 200 
http://localhost:1313/core/releases/2.13.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.13.1/ 200 
http://localhost:1313/core/releases/2.13.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.13.2/ 200 
http://localhost:1313/core/releases/2.13.3/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.13.3/ 200 
http://localhost:1313/core/releases/2.13.4/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.13.4/ 200 
http://localhost:1313/core/releases/2.13.5/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.13.5/ 200 
http://localhost:1313/core/releases/2.13.6/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.13.6/ 200 
http://localhost:1313/core/releases/2.13.7/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.13.7/ 200 
http://localhost:1313/core/releases/2.14.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.14.0/ 200 
http://localhost:1313/core/releases/2.14.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.14.1/ 200 
http://localhost:1313/core/releases/2.14.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.14.2/ 200 
http://localhost:1313/core/releases/2.14.3/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.14.3/ 200 
http://localhost:1313/core/releases/2.15.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.15.0/ 200 
http://localhost:1313/core/releases/2.16.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.16.0/ 200 
http://localhost:1313/core/releases/2.16.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.16.1/ 200 
http://localhost:1313/core/releases/2.17.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.17.0/ 200 
http://localhost:1313/core/releases/2.18.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.18.0/ 200 
http://localhost:1313/core/releases/2.18.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.18.1/ 200 
http://localhost:1313/core/releases/2.6.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.6.0/ 200 
http://localhost:1313/core/releases/2.6.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.6.1/ 200 
http://localhost:1313/core/releases/2.6.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.6.2/ 200 
http://localhost:1313/core/releases/2.6.3/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.6.3/ 200 
http://localhost:1313/core/releases/2.7.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.7.0/ 200 
http://localhost:1313/core/releases/2.7.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.7.1/ 200 
http://localhost:1313/core/releases/2.7.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.7.2/ 200 
http://localhost:1313/core/releases/2.7.3/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.7.3/ 200 
http://localhost:1313/core/releases/2.8.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.8.0/ 200 
http://localhost:1313/core/releases/2.8.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.8.1/ 200 
http://localhost:1313/core/releases/2.8.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.8.2/ 200 
http://localhost:1313/core/releases/2.8.3/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.8.3/ 200 
http://localhost:1313/core/releases/2.8.4/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.8.4/ 200 
http://localhost:1313/core/releases/2.8.5/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.8.5/ 200 
http://localhost:1313/core/releases/2.9.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.9.0/ 200 
http://localhost:1313/core/releases/2.9.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/2.9.1/ 200 
http://localhost:1313/core/releases/3.0.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.0.0/ 200 
http://localhost:1313/core/releases/3.1.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.1.0/ 200 
http://localhost:1313/core/releases/3.10.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.10.0/ 200 
http://localhost:1313/core/releases/3.10.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.10.1/ 200 
http://localhost:1313/core/releases/3.10.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.10.2/ 200 
http://localhost:1313/core/releases/3.10.3/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.10.3/ 200 
http://localhost:1313/core/releases/3.10.4/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.10.4/ 200 
http://localhost:1313/core/releases/3.10.5/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.10.5/ 200 
http://localhost:1313/core/releases/3.11.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.11.0/ 200 
http://localhost:1313/core/releases/3.11.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.11.1/ 200 
http://localhost:1313/core/releases/3.11.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.11.2/ 200 
http://localhost:1313/core/releases/3.11.3/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.11.3/ 200 
http://localhost:1313/core/releases/3.12.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.12.0/ 200 
http://localhost:1313/core/releases/3.12.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.12.1/ 200 
http://localhost:1313/core/releases/3.13.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.13.0/ 200 
http://localhost:1313/core/releases/3.14.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.14.0/ 200 
http://localhost:1313/core/releases/3.14.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.14.1/ 200 
http://localhost:1313/core/releases/3.14.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.14.2/ 200 
http://localhost:1313/core/releases/3.15.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.15.0/ 200 
http://localhost:1313/core/releases/3.16.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.16.0/ 200 
http://localhost:1313/core/releases/3.16.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.16.1/ 200 
http://localhost:1313/core/releases/3.17.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.17.0/ 200 
http://localhost:1313/core/releases/3.17.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.17.1/ 200 
http://localhost:1313/core/releases/3.17.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.17.2/ 200 
http://localhost:1313/core/releases/3.2.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.2.0/ 200 
http://localhost:1313/core/releases/3.2.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.2.1/ 200 
http://localhost:1313/core/releases/3.3.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.3.0/ 200 
http://localhost:1313/core/releases/3.4.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.4.0/ 200 
http://localhost:1313/core/releases/3.4.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.4.1/ 200 
http://localhost:1313/core/releases/3.5.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.5.0/ 200 
http://localhost:1313/core/releases/3.6.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.6.0/ 200 
http://localhost:1313/core/releases/3.6.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.6.1/ 200 
http://localhost:1313/core/releases/3.6.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.6.2/ 200 
http://localhost:1313/core/releases/3.7.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.7.0/ 200 
http://localhost:1313/core/releases/3.7.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.7.1/ 200 
http://localhost:1313/core/releases/3.8.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.8.0/ 200 
http://localhost:1313/core/releases/3.8.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.8.1/ 200 
http://localhost:1313/core/releases/3.8.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.8.2/ 200 
http://localhost:1313/core/releases/3.9.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.9.0/ 200 
http://localhost:1313/core/releases/3.9.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.9.1/ 200 
http://localhost:1313/core/releases/3.9.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/3.9.2/ 200 
http://localhost:1313/core/releases/4.0.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.0.0/ 200 
http://localhost:1313/core/releases/4.0.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.0.1/ 200 
http://localhost:1313/core/releases/4.1.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.1.0/ 200 
http://localhost:1313/core/releases/4.1.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.1.1/ 200 
http://localhost:1313/core/releases/4.1.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.1.2/ 200 
http://localhost:1313/core/releases/4.10.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.10.0/ 200 
http://localhost:1313/core/releases/4.11.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.11.0/ 200 
http://localhost:1313/core/releases/4.12.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.12.0/ 200 
http://localhost:1313/core/releases/4.13.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.13.0/ 200 
http://localhost:1313/core/releases/4.14.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.14.0/ 200 
http://localhost:1313/core/releases/4.15.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.15.0/ 200 
http://localhost:1313/core/releases/4.16.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.16.0/ 200 
http://localhost:1313/core/releases/4.17.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.17.0/ 200 
http://localhost:1313/core/releases/4.18.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.18.0/ 200 
http://localhost:1313/core/releases/4.2.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.2.0/ 200 
http://localhost:1313/core/releases/4.2.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.2.1/ 200 
http://localhost:1313/core/releases/4.2.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.2.2/ 200 
http://localhost:1313/core/releases/4.2.3/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.2.3/ 200 
http://localhost:1313/core/releases/4.2.4/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.2.4/ 200 
http://localhost:1313/core/releases/4.3.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.3.0/ 200 
http://localhost:1313/core/releases/4.3.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.3.1/ 200 
http://localhost:1313/core/releases/4.3.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.3.2/ 200 
http://localhost:1313/core/releases/4.4.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.4.0/ 200 
http://localhost:1313/core/releases/4.4.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.4.1/ 200 
http://localhost:1313/core/releases/4.4.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.4.2/ 200 
http://localhost:1313/core/releases/4.5.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.5.0/ 200 
http://localhost:1313/core/releases/4.5.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.5.1/ 200 
http://localhost:1313/core/releases/4.5.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.5.2/ 200 
http://localhost:1313/core/releases/4.6.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.6.0/ 200 
http://localhost:1313/core/releases/4.7.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.7.0/ 200 
http://localhost:1313/core/releases/4.7.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.7.1/ 200 
http://localhost:1313/core/releases/4.7.2/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.7.2/ 200 
http://localhost:1313/core/releases/4.8.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.8.0/ 200 
http://localhost:1313/core/releases/4.8.1/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.8.1/ 200 
http://localhost:1313/core/releases/4.9.0/ Is redirected! Result is:
    -> http://localhost:1313/releases/4.9.0/ 200 
http://localhost:1313/reference-apps/contact-tracing/ 404
http://localhost:1313/reference-apps/phc-covid/ 404
http://localhost:1313/reference-apps/covid-education/ 404
http://localhost:1313/reference-apps/covid-rdt-reference-app/ 404
http://localhost:1313/reference-apps/direct-to-client/ 404
http://localhost:1313/reference-apps/ebs/ 404
http://localhost:1313/building/forms/_partial_expression_functions/ 404
http://localhost:1313/reference-apps/learning-care/ 404
http://localhost:1313/reference-apps/interoperability/ 404
http://localhost:1313/reference-apps/pharmacovigilance-reference-app/ 404
http://localhost:1313/reference-apps/pih/ 404
http://localhost:1313/reference-apps/training/ 404
http://localhost:1313/reference-apps/stock-monitoring/ 404
http://localhost:1313/reference-apps/supervisor-reference-app/ 404
Successfully checked 439 URLs!

mrjones-plip reviewed (changes requested):
wow - nice work here Lorena!  Great to see all the aliases in place and I appreciate the link checker script now working on macOS \o/

Requesting a couple changes to see this through

mrjones-plip commented on .github/scripts/check.urls.sh:
```diff
@@ -56,8 +65,8 @@ if [[ ! $REPLY =~ ^[nN]$ ]]
-    url_count=$(echo "$prod_urls" | wc -l | cut -d' ' -f1)
-    echo;echo "Checking ${url_count} URLs, be patient.  Any non 200 URLs will be listed here:"
+    url_count=$(echo "$prod_urls" | tr -cd '\n' | wc -l | tr -d ' ')
+    echo "Found ${url_count} URLs to check, be patient. Any non 200 URLs will be listed here:"
```
```suggestion
    echo;echo "Found ${url_count} URLs to check, be patient. Any non 200 URLs will be listed here:"
```

mrjones-plip commented on .github/scripts/check.urls.sh:
```diff
@@ -56,8 +65,8 @@ if [[ ! $REPLY =~ ^[nN]$ ]]
+    url_count=$(echo "$prod_urls" | tr -cd '\n' | wc -l | tr -d ' ')
+    echo "Found ${url_count} URLs to check, be patient. Any non 200 URLs will be listed here:"
     run_checks "$prod_urls"
     echo "Successfully checked ${url_count} URLs!"
```
```suggestion
    echo;echo "Successfully checked ${url_count} URLs!";echo;
```

lorerod replied:
Thank you, @mrjones-plip!! I have a conflict to resolve before merging. I will work on it tomorrow first thing.

mrjones-plip reviewed (approved):
wonderful - thank you Lorena!

adding some minor formatting to space things out for legibility.  but approving as it's good to go!

lorerod commented:
Hi @mrjones-plip, I've just rebased this PR with the latest changes from the main branch to ensure everything is up to date. While you already approved this, I'd appreciate it if you could take a final look before I merge it.

mrjones-plip reviewed (approved):
lgtm!  thanks for iterating with me!
